File: src/nomanssave/cn.java
package nomanssave;

import java.io.File;
import javax.swing.filechooser.FileFilter;

class cn extends FileFilter {
   // $FF: synthetic field
   final cl fI;

   cn(cl var1) {
      this.fI = var1;
   }

   public String getDescription() {
      return "Planetary Base Backup File";
   }

   public boolean accept(File var1) {
      if (var1.isDirectory()) {
         return !var1.isHidden();
      } else {
         return var1.getName().endsWith(".pb3");
      }
   }
}

File: NMSSaveEditor/nomanssave/lower/cn.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.IO;
using System.Linq;
using System.Text;
using System.Text.RegularExpressions;
using System.Windows.Forms;
using System.Globalization;

namespace NMSSaveEditor
{

public class cn : FileFilter {
   // $FF: synthetic field
   public cl fI;

   public cn(cl var1) {
      this.fI = var1;
   }

   public string getDescription() {
      return "Planetary Base Backup File";
   }

   public bool accept(FileInfo var1) {
      if (var1.IsDirectory()) {
         return !var1.isHidden();
      } else {
         return var1.Name.EndsWith(".pb3");
      }
   }
}

}
